packages/graph/hash_graph/lib/graph/src/api/rest/mod.rs

```diff
@@ -12,12 +12,22 @@ mod property_type;
 
```

Comment:
Just shoving it here so we have a comment thread:

Is there a command-line utility we can use to validate the OpenAPI spec in automated tests and CI? It'd be good to make sure we don't accidentally invalidate it again.

I appreciate generating an OpenAPI client and using it in e2e tests might provide some confidence, so we can fall-back on that if needs be but it would be nice to validate the OpenAPI spec if we can.

Reply:
The tool i used https://github.com/OpenAPITools/openapi-generator-cli seems to be one of the popular ones. It has a `validate` subcommand which I also used in the description screenshot

Reply:
If we can throw it in the cargo make config in 10 minutes or so lets do it as part of this PR? 

Otherwise can you make an asana task for us to do so

Reply:
Sure let's go for it. WIll do after I address other comments :)

Reply:
Addressed in cb7d5629460041d6116ec11e93952333a3dd6bb5 we might want to make it separate to `test-rest-api`, but that means spinning up the backend separately in another task

Reply:
Nice! Thanks